src/app/core/entity-components/entity-utils/entity-select/entity-select.component.spec.ts

```diff
@@ -87,14 +87,12 @@ describe("EntitySelectComponent", () => {
     expect(component.loading.value).toBe(false);
   }));
 
-  it("should suggest all entities after an initial load", (done) => {
-    subscription = component.filteredEntities.subscribe((next) => {
-      expect(next.length).toBe(testUsers.length);
-      done();
-    });
+  it("should suggest all entities after an initial load", fakeAsync(() => {
     component.entityType = User.ENTITY_TYPE;
+    tick();
```

Comment:
Is the tick necessary?

Reply:
It seems to be necessary. At least this test fails without the tick.